diff --git a/src/Copy.mo b/src/Copy.mo
index f090fa8..fd54508 100644
--- a/src/Copy.mo
+++ b/src/Copy.mo
@@ -1,4 +1,4 @@
-import { min } = "mo:base-0.7.3/Nat";
+import { min } = "mo:base/Nat";
 
 module {
     public func copy<T>(dst : [var T], src : [T]) {
@@ -8,7 +8,7 @@ module {
     public func copyOffset<T>(dst : [var T], startDst : Nat, src : [T], startSrc : Nat) {
         let n = min(
             dst.size() - startDst,
-            src.size() - startSrc,  
+            src.size() - startSrc,
         );
         var i = 0;
         while (i < n) {
@@ -24,7 +24,7 @@ module {
     public func copyOffsetVar<T>(dst : [var T], startDst : Nat, src : [var T], startSrc : Nat) {
         let n = min(
             dst.size() - startDst,
-            src.size() - startSrc,  
+            src.size() - startSrc,
         );
         var i = 0;
         while (i < n) {
@@ -32,4 +32,4 @@ module {
             i += 1;
         };
     };
-};
\ No newline at end of file
+};
