diff --git a/src/Migrations.mo b/src/Migrations.mo
index 0f7486a..c2d03ba 100644
--- a/src/Migrations.mo
+++ b/src/Migrations.mo
@@ -2,7 +2,7 @@ import Debug "mo:base/Debug";
 
 import MaxBpTree "mo:augmented-btrees/MaxBpTree";
 import Cmp "mo:augmented-btrees/Cmp";
-import BTree "mo:stableheapbtreemap/BTree";
+import BTree "mo:StableHeapBTreeMap/BTree";
 import Set "mo:map/Set";
 import Nat "mo:base/Nat";
 
@@ -18,7 +18,7 @@ module {
     };
 
     public type CurrentMemoryRegion = MemoryRegionV1;
-    
+
     public func upgrade(versions: VersionedMemoryRegion) : VersionedMemoryRegion {
         switch(versions){
             // case (#empty) Debug.trap("Memory region is #empty. Needs to be initialized.");
@@ -64,7 +64,7 @@ module {
 
         var pages : Nat;
     };
-    
+
     public type FreeMemoryV1 = MaxBpTree.MaxBpTree<Nat, Nat>;
 
     /// Initial version of the memory region.
@@ -82,4 +82,4 @@ module {
     };
 
 
-};
\ No newline at end of file
+};
